pkg/ctl/functions/upload_test.go

```diff
@@ -128,6 +125,8 @@ func TestUploadAndDownloadCommands(t *testing.T) {
 		log.Fatal(err)
 		t.Fail()
 	}
+	defer os.RemoveAll(f.Name())
```

Comment:
Old behavior looks better.

Reply:
use `log.Panic` instead of `log.Fatal`, since the `log.Fatal` will skip `defer` calls.

Reply:
Good catch!